apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -0,0 +1,565 @@
+                </tr>
+                <tr>
+                  <th>QID</th>
+                  <td>
+                    <span class="mr-2">${resLocals.question.qid}</span>
+                    ${resLocals.authz_data.has_course_permission_edit &&
+                    !resLocals.course.example_course
+                      ? html`
+                          <button
+                            type="button"
+                            class="btn btn-xs btn-secondary mr-2"
+                            id="changeQidButton"
+                            data-toggle="popover"
+                            data-container="body"
+                            data-html="true"
+                            data-placement="auto"
+                            title="Change QID"
+                            data-content="${renderEjs(
+                              __filename,
+                              "<%= include('../partials/changeIdForm') %>",
+                              {
+                                id_label: 'QID',
+                                buttonID: 'changeQidButton',
+                                id_old: resLocals.question.qid,
+                                ids: qids,
+                                ...resLocals,
+                              },
+                            )}"
+                            data-trigger="click"
+                          >
+                            <i class="fa fa-i-cursor"></i>
+                            <span>Change QID</span>
+                          </button>
+                        `
+                      : ''}
+                    ${questionGHLink
+                      ? html` <a target="_blank" href="${questionGHLink}"> view on GitHub </a> `
+                      : ''}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Configuration</th>
+                  <td>
+                    ${resLocals.authz_data.has_course_permission_view
+                      ? html`
+                          <a
+                            href="${resLocals.urlPrefix}/question/${resLocals.question
+                              .id}/file_view/${infoPath}"
+                          >
+                            info.json
+                          </a>
+                          ${resLocals.authz_data.has_course_permission_edit &&
+                          !resLocals.course.example_course
+                            ? html`
+                                <a
+                                  class="btn btn-xs btn-secondary mx-2"
+                                  href="${resLocals.urlPrefix}/question/${resLocals.question
+                                    .id}/file_edit/${infoPath}"
+                                >
+                                  <i class="fa fa-edit"></i>
+                                  <span>Edit</span>
+                                </a>
+                              `
+                            : html`info.json`}
```

Comment:
We end up rendering `info.json` twice in this case:

<img width="291" alt="Screenshot 2024-04-01 at 15 22 56" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/ae791051-5140-458e-936d-6ab4b2cd656a">

I think we should just render nothing (`''`) here in this branch.

Reply:
Updated to render `''` here in [d5b67e6](https://github.com/PrairieLearn/PrairieLearn/pull/9571/commits/d5b67e64989f49d989864e2afd948f2db1b57803).